common/src/main/java/com/microsoft/hydralab/performance/inspectors/AndroidBatteryInfoInspector.java

```diff
@@ -28,8 +28,8 @@ public PerformanceInspectionResult inspect(PerformanceInspection performanceInsp
         File rawResultFile = new File(rawResultFolder,
                 String.format(RAW_RESULT_FILE_NAME_FORMAT, getClass().getSimpleName(), performanceInspection.appId, TimeUtils.getTimestampForFilename()));
 
-        ShellUtils.execLocalCommandWithResult(String.format("adb -s %s shell dumpsys batterystats %s | out-file %s -encoding utf8",
-                performanceInspection.deviceIdentifier, performanceInspection.appId, rawResultFile.getAbsolutePath()), classLogger);
+        ShellUtils.execLocalCommandWithRedirect(String.format("adb -s %s shell dumpsys batterystats %s",
+                performanceInspection.deviceIdentifier, performanceInspection.appId), rawResultFile, false, classLogger);
         return new PerformanceInspectionResult(rawResultFile, performanceInspection);
```

Comment:
I ran the code locally, and this line caused an error:
![image](https://user-images.githubusercontent.com/30514480/228760598-e3a17953-28dc-4cbb-acf6-b19af558f434.png)